adlfs/spec.py

```diff
@@ -898,7 +898,11 @@ async def _details(
                     data["size"] = None
                 elif data["metadata"].get("is_directory") == "false":
                     data["type"] = "file"
-                elif data["metadata"].get("hdi_isfolder") == "true":
+                elif (
+                    # In some cases Hdi_isfolder is capitalized, see #418
+                    data["metadata"].get("hdi_isfolder") == "true"
+                    or data["metadata"].get("Hdi_isfolder") == "true"
```

Comment:
```suggestion
                    data["metadata"].get("hdi_isfolder").lower() == "true"
```

Do we worry about other miscapitalizations?

Reply:
Thanks @TomAugspurger for taking a look here! I do not think that your suggested change is doing the same and it wouldn't work.

Reply:
Sounds good, let's ignore that then.